README.md

```diff
@@ -4,20 +4,91 @@ This project is a ActiveMQ Artemis Self Provisioning Plugin to the Administrator
 
 ## Local development
 
+To be able to run the local development environment you need to:
+* have access to an OpenShift cluster
+* have the operator installed on the cluster
+* have the plugin running
+* have the console running
+
+### Setting up an OpenShift cluster
+
+In order to run the project you need to have access to an OpenShift cluster.
+You can either request one to the `Cluster bot` on slack, or install one on your
+local development environment.
+
+#### Remote cluster
+
+Find the `Cluster bot` app on slack and send it a message to start a cluster
+  `launch 4.12`
+
+Starting the cluster can take between 15 and 45 minutes. Once it is up it'll
+stay up for about 190 minutes.
+
+> [!IMPORTANT]
+> for connecting to the cluster you will need to have access to the `oc`
+> command. One way to get it is to follow the local cluster setup as it will pull
+> the binary for you. As far as the author knows the `oc` binary can't be
+> installed with traditional packages.
+
+#### Local cluster
+
+Follow the documentation:
+https://access.redhat.com/documentation/en-us/red_hat_openshift_local/2.34/html-single/getting_started_guide/index#introducing
+
+> [!WARNING]
+> If you're running the cluster on Fedora CSB, note that [you might
+> need](https://github.com/crc-org/crc/issues/4110) to
+> configure the network as local: `crc config set network-mode user`
+
+Once your environment is set up you simply need to `crc start` your cluster.
+
+#### Connecting to the cluster
+
+Depending on the remote or local env:
+
+* `oc login -u kubeadmin
+https://api.ci-ln-x671mxk-76ef8.origin-ci-int-aws.dev.rhcloud.com:6443` (to
+adapt depending on your cluster address)
+* `oc login -u kubeadmin https://api.crc.testing:6443`
+
+### Installing the operator
```

Comment:
In openshift we can use the operaterHub to install the operator. unless you want to test against the upstream operator. Would be nice to add the operatorHub option here.

Reply:
When testing I wasn't successful at installing through the operator hub:

On a cluster launched via `Cluster bot` I had this issue:

![image](https://github.com/artemiscloud/activemq-artemis-self-provisioning-plugin/assets/86971992/24e3161a-f841-44de-a8a4-ac1a5470f36a)

On my local cluster there nothing under the operator hub:

![image](https://github.com/artemiscloud/activemq-artemis-self-provisioning-plugin/assets/86971992/99f66611-3e1d-470e-9f6e-f3880e1aa302)

Do you know about extra steps I must do to have acces to the hub on my local machine?

Reply:
for your local you should check the pods state
`oc get pods -n openshift-marketplace` see if all the pods are running state. Sometimes it takes a while to get those pods ready.

Reply:
```
NAME                                    READY   STATUS             RESTARTS        AGE
certified-operators-tvk2q               0/1     CrashLoopBackOff   923 (28s ago)   20d
community-operators-z9bzg               0/1     CrashLoopBackOff   921 (28s ago)   20d
marketplace-operator-568756cbf5-654nq   1/1     Running            4               21d
redhat-marketplace-zf49d                0/1     CrashLoopBackOff   923 (29s ago)   20d
redhat-operators-9jb87                  0/1     CrashLoopBackOff   918 (28s ago)   20d
```
Ok, something is wrong with the marketplace, I need to investigate what is going on!

Reply:
I followed this issue resolution and it fixed my problem https://github.com/crc-org/crc/issues/4109

Reply:
I've added the related piece of information in the documentation